gccg_transport_api.h

```diff
@@ -42,7 +42,15 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
   ## Values that are valid for both source and destination connections. ##
   "profile_version": "01.00", ## Version of this JSON ##
   "protocol": "cdi",          ## TODO Other types: "rtp", "tcp", "ndi", "srt", "socket", "other". Platform specific? ##
-  "bandwidth": "14000000",    ## Maximum required bandwidth for the connection. ##
+  "bandwidth": 14000000,      ## Maximum required bandwidth for the connection. ##
+
+  "timing": {         ## Note: These values should not change over the lifetime of the connection.
+    "GMID": 12345678, ## 64-bit Grandmaster Clock Identifier ##
+    "COT": 1234,      ## Content Origination Timestamp ##
+    "LAT": 1234,      ## Local Arrival Timestamp ##
+    "Tmin": 100,      ## Minimum latency of the Workflow Step ##
+    "T99": 200        ## Maximum latency of the Workflow Step ##
```

Comment:
what units?

Reply:
Will add to COT and LAT that they are 64-bit values and use something like: Upper 32-bits is the number of seconds since the SMPTE Epoch. Lower 32-bits is the number of fractional seconds as measured in nanoseconds.

Will add to Tmin and T99 that values are in milliseconds.